spec/features/checkouts_spec.rb

```diff
@@ -1,9 +1,23 @@
 require 'rails_helper'
 
 RSpec.feature "Checkouts", type: :feature do
+
+  before(:each) do
+  O1 = Order.create(Status: "Completed", total: 23600,
+                   vat: 2100, delivery_cost: 500, user_id: 6, 
+                   created_at: "2015-10-20 15:49:28", updated_at: "2015-10-20 15:49:28", 
+                   transaction_id: "1VX775486T314881H", invoice: "", pickup_time: 26)
+  
+  O2 = Order.create(Status: "Completed", total: 23600, 
+                    vat: 2100, delivery_cost: 500, user_id: 6,
+                    created_at: "2015-10-20 15:49:28", updated_at: "2015-10-20 15:49:28", 
+                    transaction_id: "1VX775486T314881H", invoice: "", pickup_time: 26)
+  end
+
   describe "User can check out from Carts page" do
     scenario "user clicks checkout button on carts page" do
       Food.create(name: "coconut rice", description: "tasty coconut rice cooked with shrimps", price: 4000)
+      Order.create(Status: "Completed", total: 23600, vat: 2100, delivery_cost: 500, user_id: 6, created_at: "2015-10-20 15:49:28", updated_at: "2015-10-20 15:49:28", transaction_id: "1VX775486T314881H", invoice: "", pickup_time: 26)
```

Comment:
Line is too long. [233/80]<br>Separate every 3 digits in the integer portion of a number with underscores(_).